src/main/java/com/google/ehub/data/CommentData.java

```diff
@@ -8,11 +8,17 @@
   private final String comment;
   private final Long timestampMillis;
   private final String username;
+  private final Long commentId;
+  private final String commentKind;
+  private final Boolean belongsToUser;
 
-  public CommentData(long itemId, String comment, long timestampMillis, String username) {
+  public CommentData(long itemId, String comment, long timestampMillis, String username, long commentId, String commentKind, boolean belongsToUser) {
```

Comment:
Is this page formatted? I think the constructor might be over 80 lines. Please run clang-format to make sure.